lib/elixir/lib/range.ex

```diff
@@ -461,6 +461,69 @@ defmodule Range do
     end
   end
 
+  @doc """
+  Checks if all points of range2 are inside range1.
```

Comment:
Please use Markdown backticks on the argument names, so they'll render nicely in the documentation 🙃 Applies to the whole `@doc` here.

```suggestion
  Checks if all points of `range2` are inside `range1`.
```

Reply:
done!